irods_testing_environment/irods_config.py

```diff
@@ -383,14 +383,29 @@ def configure_irods_federation_testing(ctx, remote_zone, zone_where_tests_will_r
     execute.execute_command(container, 'iadmin lu', user='irods')
     execute.execute_command(container, 'iadmin lz', user='irods')
 
-    # create zonehopper user
+    # create zonehopper#<local_zone> user
     username = '#'.join(['zonehopper', zone_where_tests_will_run.zone_name])
     mkuser = 'iadmin mkuser {} rodsuser'.format(username)
     logging.info('creating remote user [{}] [{}]'.format(mkuser, container.name))
     if execute.execute_command(container, mkuser, user='irods') is not 0:
         raise RuntimeError('failed to create remote user [{}] [{}]'
                            .format(username, container.name))
 
+    # create zonehopper#<remote_zone> user
+    username = '#'.join(['zonehopper', remote_zone.zone_name])
+    mkuser = 'iadmin mkuser {} rodsuser'.format(username)
+    logging.info('creating remote user [{}] [{}]'.format(mkuser, container.name))
```

Comment:
I wonder if the word "remote" should be removed or qualified in some way. This is a *local* user in the *remote* zone. The log message on 389 is not much better -- it is a *remote* user in the *remote* zone, where the zone of the remote user is the *local* zone.

Reply:
gah.    yes, please let's find some clean words.

Reply:
I think removing "remote" and adding "on/in container" between "[{}] [{}]" makes things clear.

For example:
```bash
'creating user [{}] in container [{}]' 
```

Thoughts?

Reply:
Looks good to me. My log messages are definitely writer-optimized in this repo lol

Reply:
Ok. I'll move forward with that.

Reply:
Wait, it should read something like ... `executing command [{}] in container [{}]`

See the arguments being passed to `.format()`.

Is it better to see the command that is being executed or to swap out the first argument with the user's full name?

I think logging the command is better for debugging purposes.

Reply:
Well, `execute.execute` will log commands at level `INFO` or higher (i.e. `-vv`) anyway, so I don't think adding it again here will be particularly helpful

Reply:
Although, I don't think *that* log message is logging the executing container. So, maybe. Whatever looks most helpful for you is probably fine

Reply:
You're saying showing the username is better here?

Reply:
I adjusted the message to log the user's qualified name rather than the command to execute.

Let me know what you think.

Reply:
Perfect. Thanks!

Reply:
Done.